app/src/main/res/layout/fragment_file_list.xml

```diff
@@ -225,7 +227,8 @@
             <androidx.core.widget.NestedScrollView
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
-                android:fillViewport="true">
+                android:fillViewport="true"
+                android:touchscreenBlocksFocus="true">
```

Comment:
touchscreenBlocksFocus is also needed for nested scroll views? Or maybe rather, it's needed on the collapsing toolbar layout?